src/BitMapViewer.cpp

```diff
@@ -177,31 +177,25 @@ void BitMapViewer::refresh()
 void BitMapViewer::on_screenMode_currentIndexChanged(int index)
 {
 	screenMod = screenMode->itemText(index).toInt();
-	printf("\nnew screenMod: %i\n", screenMod);
-	//change then number of visibe pages when this changes
+	pageSize = screenMod >= 7 ? 65536 : 32768;
+
+	// change then number of visibe pages when this changes
```

Comment:
I know you didn't change this comment. But should it maybe say "change *the* number ..."?